src/PluginClient.php

```diff
@@ -110,6 +111,7 @@ private function configure(array $options = [])
         $resolver = new OptionsResolver();
         $resolver->setDefaults([
             'max_restarts' => 10,
+            'debug_plugins' => [],
```

Comment:
we can use a callback, as in this example:
https://github.com/symfony/symfony/issues/15524#issuecomment-130369403